articles/app-service-mobile/app-service-mobile-windows-store-dotnet-get-started-users.md

```diff
@@ -27,6 +27,20 @@ This tutorial is based on the Mobile Apps quickstart. You must first complete th
 ## <a name="register"></a>Register your app for authentication and configure the App Service
 [!INCLUDE [app-service-mobile-register-authentication](../../includes/app-service-mobile-register-authentication.md)]
 
+## <a name="redirecturl"></a>Add your app to the Allowed External Redirect URLs
+
+Secure authentication requires that you define a new URL scheme for your app. This allows the authentication system to redirect back to your app once the authentication process is complete. In this tutorial, we use the URL scheme _appname_ throughout. However, you can use any URL scheme you choose. It should be unique to your mobile application. To enable the redirection on the server side:
```

Comment:
content says "any URL scheme you choose", however Section 7.1 of the OAuth 2 spec for native apps (https://tools.ietf.org/html/draft-ietf-oauth-native-apps-12#section-7.1) suggests:
 
>“When choosing a URI scheme to associate with the app, apps MUST use a
   URI scheme based on a domain name under their control, expressed in
   reverse order, as recommended by Section 3.8 of [RFC7595] for
   private-use URI schemes.”

It's not a huge problem to leave the text 'as is', however customers could inadvertently choose a poor scheme name without this additional context?

Some providers (e.g. Google) force you do this, in that they give you the URL scheme to use.